### ISSUE TITLE
Window size overlay does not go away

### ISSUE THREAD

**Issue by xinan** [blinksh/blink#407]
I often encounter this problem where the overlay that shows the window size, the one that looks like:
```
-----------
|         |
|  Blink  |
| 51 x 70 |
|         |
-----------
```
stays on the screen and does not go away when I switch back to Blink from other apps. 

It does not happen all the time, but it happens quite often, especially when it is in split screen mode.

I am using an 1st gen iPad Pro 12.9, running iOS 11.2 (it happens when I was using iOS 10 too), with the Apple Smart Keyboard.

**Comment by carloscabanero** [comment]
I still see this one once in a blue moon. We patched MBProgressHUD (jdg/MBProgressHUD#440) but looks like not all cases. They have updated the project, so we should do the same and see if that does the trick.

**Comment by carloscabanero** [comment]
Dupe of #385. Closing here
